Tests/Functional/TestBase.php

```diff
@@ -50,41 +100,13 @@ public function importData($file)
     }
 
     /**
-     * Sets up required info before each test.
-     */
-    protected function setUp()
-    {
-        AnnotationRegistry::registerFile(
-            'vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/Driver/DoctrineAnnotations.php'
-        );
-        /** @var EntityManager $entityManager */
-        $entityManager = $this->getEntityManager();
-        $connection = $entityManager->getConnection();
-        $params = $connection->getParams();
-        $name = $connection->getParams()['dbname'];
-        unset($params['dbname']);
-        $tmpConnection = DriverManager::getConnection($params);
-        $name = $tmpConnection->getDatabasePlatform()->quoteSingleIdentifier($name);
-        try {
-            $tmpConnection->getSchemaManager()->dropDatabase($name);
-        } catch (\Exception $ex) {
-            // Catching exception in case database not dropped.
-        }
-        $tmpConnection->getSchemaManager()->createDatabase($name);
-        $tmpConnection->close();
-    }
-
-    /**
-     * Deletes  all data after each test.
+     * Return full route to db file.
+     *
+     * @return string
      */
-    protected function tearDown()
+    public function getSetUpDbFile()
     {
-        /** @var EntityManager $entityManager */
-        $entityManager = $this->getEntityManager();
-        $connection = $entityManager->getConnection();
-        $name = $connection->getParams()['dbname'];
-        $name = $connection->getSchemaManager()->getDatabasePlatform()->quoteSingleIdentifier($name);
-        $connection->getSchemaManager()->dropDatabase($name);
+        return null;
```

Comment:
if you need setUp defaul fault you can extend this method. For example in magentoBundle:

/**
     * {@inheritdoc}
     */
    public function getSetUpDbFile()
    {
        return '/fixtures/magento_db.sql';
    }